### PR TITLE
Add 413 response error for produce v3 api

### PR DESCRIPTION
**Pull request by trnguyencflt**
Follow up the PR https://github.com/confluentinc/rest-utils/pull/411, this is to update API doc for produce records v3 API in kafka-rest

### CODE REVIEW

**Comment by trnguyencflt** [comment]
@tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍

**Comment by tobydrake7** [comment]
> @tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍

@trnguyencflt I believe this update to the API spec is all that is necessary, since it's imported into the public docs and is an error response description.

File: api/v3/openapi.yaml [hunk 1]
```diff
@@ -4493,6 +4495,20 @@ components:
                         </body>
                     </html>'
 
+    RequestEntityTooLargeErrorResponse:
+      description: 'This implies client is sending the request payload that is larger than the max message size the
+        server will accept.'
```
Comment (tobydrake7) [inline api/v3/openapi.yaml:4500]:
```suggestion
      description: 'This implies the client is sending a request payload that is larger than the maximum message size the
        server can accept.'
```

File: api/v3/openapi.yaml [hunk 2]
```diff
@@ -4493,6 +4495,20 @@ components:
...
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/Error'
+          examples:
+            produce_records_expects_json:
+              description: "Thrown by /records API if payload size exceeds the message max size"
+              value:
+                error_code: 413
+                message:  "The request included a message larger than the max message size the server will accept."
```
Comment (tobydrake7) [inline api/v3/openapi.yaml:4510]:
```suggestion
                message:  "The request included a message larger than the maximum message size the server can accept."
```

File: api/v3/openapi.yaml [hunk 3]
```diff
@@ -4493,6 +4495,20 @@ components:
...
+
     UnsupportedMediaTypeErrorResponse:
       description: 'This implies client is sending the request payload format in an supported format.'
```
Comment (tobydrake7) [inline api/v3/openapi.yaml:4513]:
```suggestion
      description: 'This implies the client is sending the request payload in a supported format.'
```

File: api/v3/openapi.yaml (hunk 1 above)
Comment (trnguyencflt) [reply api/v3/openapi.yaml]: done

File: api/v3/openapi.yaml (hunk 2 above)
Comment (trnguyencflt) [reply api/v3/openapi.yaml]: done

File: api/v3/openapi.yaml (hunk 3 above)
Comment (trnguyencflt) [reply api/v3/openapi.yaml]: done

**Comment by trnguyencflt** [comment]
> > @tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍
> 
> @trnguyencflt I believe this update to the API spec is all that is necessary, since it's imported into the public docs and is an error response description.

@tobydrake7 I've addressed your comments, could you please have a look again? thanks 👍

File: api/v3/openapi.yaml [hunk 4]
```diff
@@ -4493,8 +4495,22 @@ components:
...
+          examples:
+            produce_records_expects_json:
+              description: "Thrown by /records API if payload size exceeds the message max size"
+              value:
+                error_code: 413
+                message:  "The request included a message larger than the maximum message size the server can accept."
+
     UnsupportedMediaTypeErrorResponse:
-      description: 'This implies client is sending the request payload format in an supported format.'
+      description: 'This implies the client is sending the request payload format in an unsupported format.'
```
Comment (msn-tldr) [inline api/v3/openapi.yaml:4513]: good catch

**Review by msn-tldr — APPROVED** [review]
lgtm

File: api/v3/openapi.yaml (hunk 4 above)
Comment (trnguyencflt) [reply api/v3/openapi.yaml:4513]: all credit to @tobydrake7 😄

**Pull request merged**